homeassistant/bootstrap.py

```diff
@@ -408,6 +409,10 @@ def log_exception(ex, domain, config):
         message += " (See {}:{})".format(config.__config_file__,
                                          config.__line__ or '?')
 
+    if domain != 'homeassistant':
+        message += ('. Please check the docs at '
```

Comment:
You will an extra period if `extra keys not allowed` is NOT in `ex.error_message`